book-example/src/cli/clean.md

```diff
@@ -0,0 +1,22 @@
+# The clean command
+
+The `clean` command is used to delete directory where built book is located.
+
+```bash
+mdbook clean
+```
+
+It will try to delete built book. If you pass path to directory it will be used
+instead.
+
+
+#### Specify a directory
+
+Like `init`, the `clean` command can take a directory as an argument to use
+instead of the standard directory where generated book is placed once built.
```

Comment:
This second line sounds a little funny to me. Perhaps you could say

```diff
- instead of the standard directory where generated book is placed once built.
+ instead of the normal build directory.
```

Reply:
Changed.